### DISCUSSION TITLE
Access token is require which is missing

### DISCUSSION THREAD

**Discussion by gpoveda** [mynaparrot/plugNmeet-server#405] [General]
Hi, 

I'm sorry, I would like to know how can I fix the issue related to "Access token is require which is missing" when I try to access after a fresh installation.

Everything in server (docker) is well: 

db
etherpad
livekit
plugnmeet-api
livekit-ingress
server-redis 

Also in client:

webpack 5.89.0 compiled successfully in 83542 ms
No errors found.

Im not sure if I need to include some "key" from config server in some file from the client files ?

I really appreciate all your help.

Thanks

**Comment by jibon57** [comment]
Please follow: https://www.plugnmeet.org/docs/api/intro
Or to direct access login page: https://your-pnm.com/login.html

**Reply by smvelo** [reply]
Bro, I am still having the issue: "Access token missing , Access token is required which is missing" I appreciate some help please. I have the token already set on config.yaml but for sone reason the error is unresolved and I am running the project on local machine.

**Comment by gpoveda** [comment]
Hi @jibon57 

Thank you very much for your quick response. I have already solve it thanks your advise. Im sorry , I have another question: do you know how can I set up SSL on a docker installation. I can see you have previously explained for stand alone installation with haproxy. Im using this --> https://www.plugnmeet.org/docs/developer-guide/setup-development/

P.S: I dont know is better move this message to the discussion session.


Thank you very much

**Reply by jibon57** [reply]
@gpoveda I've moved to discussion.

Using a load balancer or proxy in front of plugNmeet is recommended. This way you'll be able to reduce complexity. You can have a look installation script to get better ideas.

**Comment by smvelo** [comment]
I have the same token issue however the config.yml on server and client side they have the same token

**Reply by jibon57** [reply]
@smvelo you'll need to generate token by API: https://www.plugnmeet.org/docs/api/intro
Or if you want to access server directly then: https://your_plugnmeet_server/login.html

**Comment by smvelo** [comment]
How to generate it and to which filet to add it ?
________________________________

**Comment by smvelo** [comment]
Alright got the login page but now logging in is not working
________________________________

**Reply by waelelfeqy** [reply]
me too. Still login page not loading.
